test/integration/PaymentTypes/ApplepayTest.php

```diff
@@ -407,14 +407,14 @@ public function applepayChargeCanBeCanceled(): void
-          "header": {
-            "ephemeralPublicKey": "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEW7hYAxjCeE/r9SSRX/hJsfO+VxLvUqIzyeGn6lZ1v/pYYS66Bz0dsSzoMMZg8G32TAPXUr97AD4zCXfcQoZaOA==",
-            "publicKeyHash": "zqO5Y3ldWWm4NnIkfGCvJILw30rp3y46Jsf21gE8CNg=",
-            "transactionId": "94f6b37149ae2098efb287ed0ade704284cff3f672ef7f0dc17614b31e926b9d"
-          }
+            "version": "EC_v1",
```

Comment:
Shouldn't this be generated on Test run?
Why showing this to the merchant?
Is it safe to share all the information here?

Reply:
I can not generate this on testrun like that, because it would requires me to use an apple device to get those payment data.
These are only testdata, so it should not be crusual at that point